fusesoc/capi2/core.py

```diff
@@ -638,3 +640,50 @@ def get_description(self):
     @property
     def mapping(self) -> Optional[Mapping[str, str]]:
         return MappingProxyType(self._coredata.get("mapping", {}))
+
+    def sig_status_long(self, trustfile):
+        if not trustfile:
+            siginfo = "No trustfile"
+        elif not os.path.isfile(trustfile):
+            siginfo = "Configured trustfile does not exist"
+        else:
+            match self.sig_status(trustfile):
```

Comment:
IIUC match seems to only be available from Python 3.10. If so, please use regular if else instead